### ISSUE TITLE
Use baseline instead of progressive jpg encoding

### ISSUE THREAD

**Issue by pgraci** [ysugimoto/aws-lambda-image#134]
Both `mozjpeg` and `jpegoptim` are outputting progressive jpgs.  Is it possible to override this to default to baseline or provide an option to do so?

i see that it is possible to edit https://github.com/ysugimoto/aws-lambda-image/blob/39fa9f96f7781860a136bc132e4aefee21eba10e/lib/optimizer/Mozjpeg.js#L17 and change `-progressive` to `-baseline`.  Could this option be passed in via config?

**Comment by ysugimoto** [comment]
Sounds good.

I've planned to add a new parameter in configuration in order to pass to the `Optimizers` like this:

```
{
  ...
  "optimizers: {
    "mozjpeg": ["-optimize", "-progressive"],
    "pnguant": [...]
}
```
It this configuration was supplied, override its command arguments.
Is this appropriate to you?

**Comment by pgraci** [comment]
Yes @ysugimoto that would be great.  I have manually changed the values for now, but this config option would be a better solution.

**Comment by ysugimoto** [comment]
@pgraci https://github.com/ysugimoto/aws-lambda-image/pull/140 We just implemented this feature and released `v0.14.0`, plz check it out.
